### ISSUE TITLE
Predictions during training are zero

### ISSUE THREAD

**Issue by germanjke** [dropreg/efficient_alpaca#8]
In both Megatron and Megatron LoRA my gradient norm during training is 0, all predictions are zeros. Maybe some problem with weights initialization or gradient accumulation?

**Comment by germanjke** [comment]
Interesting, that not Megatron version working fine
